deps/rabbitmq_management/priv/www/js/main.js

```diff
@@ -657,21 +657,47 @@ function postprocess() {
         });
 
     $('#download-definitions').on('click', function() {
-            var idx = $("select[name='vhost-download'] option:selected").index();
-            var vhost = ((idx <=0 ) ? "" : "/" + esc($("select[name='vhost-download'] option:selected").val()));
-        if (oauth.enabled) {
-            var path = 'api/definitions' + vhost + '?download=' +
-                esc($('#download-filename').val()) +
-                '&token=' + oauth.access_token;
+        // https://stackoverflow.com/questions/16086162/handle-file-download-from-ajax-post/23797348
+        // https://gist.github.com/zynick/12bae6dbc76f6aacedf0/
+        var idx = $("select[name='vhost-download'] option:selected").index();
+        var vhost = ((idx <=0 ) ? "" : "/" + esc($("select[name='vhost-download'] option:selected").val()));
```

Comment:
This is truly one of the oddest drive-by comments I've seen.

The `(idx <=0 )` code, with it's extra space character, has been there for seven years (since https://github.com/rabbitmq/rabbitmq-server/commit/0dd068391420c70f683d3c74ee02f8f159a3dafc).

Please feel free to open a PR to remove the space character.